dkandalintsev/login.feature

```diff
@@ -0,0 +1,127 @@
+Feature: 1. Signing in with Google account
+
+Scenario: 1.01 Sign in page is available for user
+    Given user is on the https://qa.ddso-spot.quantori.com/ page
+    When user clicks "Sign in with Google" button
+    Then https://accounts.google.com/ authorization page opens
+
+Scenario: 1.02 User has one saved Google account on device and tries to sign in
+    Given user is signed in with one Google account on google.com in the web browser
+    And users's Google accounts have this credentials:
+        | email address               | phone number | password |
+        | testkandalintsev@gmail.com  | +79819139630 | Test123! |
+    And user is on the https://qa.ddso-spot.quantori.com/ page
+    When user clicks "Sign in with Google" button
+    Then user is on the Quantori Scoring main page
+
+Scenario: 1.03 User has multiple saved Google accounts on device and tries to sign in
+    Given user is signed in with two Google accounts on google.com in the web browser
+    And users's Google accounts have this credentials:
+        | email address               | phone number | password |
+        | testkandalintsev@gmail.com  | +79819139630 | Test123! |
+        | testkandalintsevd@gmail.com | +79646149307 | Test123! |
+    And user is on the https://qa.ddso-spot.quantori.com/ page
+    When user clicks "Sign in with Google" button
+    Then list of user's existing Google accounts is shown
+
+Scenario: 1.04 User signs in with multiple saved and logged in Google accounts
+    Given user has logged in Google accounts in browser
+    And user is on the https://qa.ddso-spot.quantori.com/ page
+    And user clicked "Sign in with Google" button
+    When user clicks on one of his Google accounts in the list
+    Then user is on the Quantori Scoring main page
+
+Scenario: 1.05 User has saved Google account, but signed out
+    Given user is logged out of his Google account
```

Comment:
Also, need to provide test data for the "Given" section as you used in the scenarios above.
This comment is valid for the similar mistakes below.